client/src/styles/legacy/_style.scss

```diff
@@ -708,7 +708,46 @@ html, body {
   width: 100%;
   height: 100%;
   z-index: 9999;
-  background: url("../images/spinner.gif") no-repeat 50% 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .spinner {
+    width: 3rem;
+    height: 3rem;
+    padding: .5rem;
+
+    background-color: $white;
+    border-radius: $border-radius-sm;
+
+    &__animation {
+      width: 100%;
+      height: 100%;
+
+      &__empty {
```

Comment:
Should these be `--` modifiers rather than `__` elements? Seems a bit odd to have `.spinner__animation__empty` but it might be me and my rudimentary understanding of BEM conventions

Reply:
Bump @lukereative, we could get this PR into the next release

Reply:
It's a bit ambiguous for this, because these are seperate elements that represent different parts of the logo – they're not 2 states of the same element, so I didn't make them modifiers `--`.